tests/api_test.py

```diff
@@ -5580,9 +5583,9 @@ def scanned_f(c, x):
       return out, out
 
     def f(xs):
-      return lax.scan(scanned_f, 1., xs)
+      return lax.scan(scanned_f, jnp.array(1., 'float32'), xs)
```

Comment:
Why is this an issue now?

Reply:
Great q. It's because old remat was fast and loose with weak types, in a way that new remat noticed.